src/commands/model-derivative.ts

```diff
@@ -25,7 +26,8 @@ function urnify(id: string): string {
 	return _urnify(id).replace('/', '_');
 }
 
-export async function translateObject(object: IObject | undefined, context: IContext) {
+//hi.IVersion
```

Comment:
What's the purpose of this comment?

Reply:
nothing good catch will remove :)

Reply:
Ok found a way by having 2 command definitions with different when clauses.